web/profiles/custom/yalesites_profile/modules/custom/ys_themes/src/ThemeSettingsManager.php

```diff
@@ -161,8 +161,36 @@ class ThemeSettingsManager {
         ],
       ],
     ],
+    'animation_style' => [
+      'name' => 'Animation Style',
+      'prop_type' => 'element',
+      'selector' => '[data-site-animation]',
+      'default' => 'minimal',
+      'values' => [
+        'minimal' => [
+          'label' => 'Minimal',
+        ],
+        'artistic' => [
+          'label' => 'Artistic',
+        ],
+      ],
+    ],
   ];
 
+  /**
+   * A string to represent a false value in an HTML attribute.
+   *
+   * @var string
+   */
+  const ATTRIBUTE_FALSE = "['false']";
```

Comment:
Asking Joe a question about this on team. Leaving a comment so that I can link to it.

Reply:
Sure.  I was able to look at the tracing of the == operator parsing, and it looks like it is indeed checking against "true":

<img width="723" alt="Screenshot 2023-09-01 at 10 40 09 AM" src="https://github.com/yalesites-org/yalesites-project/assets/128765777/572cb740-0f61-4410-879c-3eda47b11a4d">

In the last one, false is the overall == operator result that's being pushed onto the stack.  The first value is the lefthand value, followed by the righthand value.

Keep in mind this data is from Storybook/CL, not directly from Atomic, which would be better.  I haven't yet figured out how to get the logs out of that yet to see what we're passing from the Drupal side.

I'm going to attempt to write a quick test for this if I can between Drupal and CL to see if it keeps this when json_encoding.  But that's what I've found so far.  Let me know what you two find out as well.  :D